src/main/util.ts

```diff
@@ -84,3 +85,32 @@ export const isDebug = () => process.env.NODE_ENV === 'development' || process.e
 
 export const appdataPath = app.getPath('userData');
 export const logsPath = app.getPath('logs');
+
+
+export const elevatedPSCommand = (command: string): Promise<string> => {
+    return new Promise((resolve, reject) => {
+        exec(`powershell -Command "& {Start-Process powershell -windowstyle hidden -Verb RunAs -ArgumentList '-Command ${command}'}"`, (error: any, stdout: any, stderr: any) => {
```

Comment:
`command` needs to be escaped. Honestly I'm a little wary of doing any sort of string interpolation in an elevated PowerShell command because as soon as user-provided input gets used here it'll be trivial to circumvent, but I can get over it for now

Reply:
The hope is that we can get the whole application running in admin, so this is no longer needed and we don't have to rely on the user clicking "yes" to allow an admin operation (#32).  However, this works for now.